batch_processor.py

```
@@ -160,30 +164,59 @@ def _load_prompt(self, filename: str) -> str:
+                self._wait_for_api_interval()
+                
                 result = api_func()
+                self.last_api_call_time = time.time()
+                
                 if result and result.strip():
                     return result.strip()
                 else:
                     print(f"API返回空内容 (尝试 {attempt + 1}/{max_retries})")
                     if attempt == max_retries - 1:
                         return None
             except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)